specs/NewWindowSourceFrameInfo.md

```diff
@@ -0,0 +1,113 @@
+Source Frame Info for New Window Requested
+===
+
+# Background
+Currently there is no way to determine the source frame of a new window request. This information
+can be useful when deciding how to open the content. For example, you may want to open requests
+that originate in a 3rd party frame in the default browser instead of in a new WebView. The
+WebView2 team is extending `NewWindowRequestedEventArgs` with a `SourceFrameInfo` property to make this easier. Here we described the updated API.
```

Comment:
FrameInfo is being extended with additional properties FrameId, ParentFrameInfo, FrameKind: https://github.com/MicrosoftEdge/WebView2Feedback/pull/3675/files

Hopefully this doesn't cause any issue